### DISCUSSION TITLE
Estimating Rt

### DISCUSSION THREAD

**Discussion by cbird808** [neherlab/covid19_scenarios#818] [General]
I'm interested in estimating Rt in c19 scenarios.  My question is, do I have to account for the proportion of the population that has been infected?  

Put another way, does the Rt parameter describe transmission to only the susceptible or does it include potential transmissions to those that have been infected. In the former case, presumably the operator would have to modulate Rt as more people became infected.  In the latter case, the Rt would have to be modified to represent transmission to the susceptible.

Example. Let's say that `Rt` = 2 based upon the `Ro` and interventions, and the Rt plot output by c19 scenarios says `Rt` = 2.0.  Further, 20% of the population has been infected in the model at time `t`. Is the actual estimated `Rt` = 2.0 * (1 - 0.2) = 1.6?  or 2.0?

**Comment by rneher** [comment]
This is currently a bit ad-hoc. Rt is estimated from data into not accounting for seroprevalence. This is then converted into mitigations that generate an Rt trajectory assuming a seroprevalence such that Rt=1 corresponds to an epidemic of constant size.  

But we only use one point estimate of seroprevalence, no time dependence.

**Reply by cbird808** [reply]
Thank you for your response!  Either I'm not understanding your response completely, or perhaps I am not being 100% clear in what I'm trying to do.

My goal is to estimate the average number of transmissions per carrier at time `t`, which is not the same as the Rt output in the html graph by c19 scenarios (that is just what I've set the variable Rt to be based on Ro and mitigations, and it doesn't change unless I change it).   My original idea is in the OP.  Alternatively I could try to estimate the true Rt based on the average period of infectiousness, the number of infectious, and new infections. Do you have any other ideas or would one of these work better than the other given you understanding of how the model works? Perhaps I need to fish the "Rt trajectory" out of the code?

any assistance you can provide is much appreciated

**Reply by cbird808** [reply]
I decided to calculate Rt from the model output.  

`NewInfections(t) = Susceptible(t-1) - Susceptible(t)`
`NewInfectionsPerInfectious(t) = NewInfections(t) / Infectious(t)`
`Est_R(t) = InfectiousPeriodDays * NewInfectionsPerInfectious(t)`

If I lag the calculation of `InfectionsPerInfectious(t) = NewInfections(t) / Infectious(t-1)` the `Daily_Est_R(t)` is higher than the Rt value I'm inputting with Ro and the interventions.

I welcome any feedback
